packages/celotool/src/lib/oracle.ts

```diff
@@ -133,8 +135,9 @@ export async function removeHelmRelease(celoEnv: string, context: OracleAzureCon
 async function helmParameters(celoEnv: string, context: OracleAzureContext, useForno: boolean) {
   const oracleConfig = getOracleConfig(context)
 
-  const kubeAuthTokenName = await rbacAuthTokenName(celoEnv)
   const replicas = oracleConfig.identities.length
+  const kubeServiceAccountSecretName = await rbacAuthTokenName(celoEnv, replicas)
```

Comment:
could you change some of these names to reflect that they're the secret names and arrays?

eg
`kubeServiceAccountSecretName` -> `kubeServiceAccountSecretNames`
`rbacAuthTokenName` -> `rbacServiceAccountSecretNames`
`kube.serviceAccountSecretName` -> `kube.serviceAccountSecretNames`